### DISCUSSION TITLE
script.outbound-links.js or plausible.outbound-links.js ?

### DISCUSSION THREAD

**Discussion by stremshs** [plausible/analytics#2039] [Anything Else]
Hi everyone, 

I'm new to Plausible and I'm impressed by the tool and the community !

I've ran through different feeds :
https://github.com/plausible/analytics/discussions/1201
https://github.com/plausible/analytics/issues/1204
But still can't figure what's the main issue...

### Here is the thing : 
I wanted to get Plausible proxied on a shared hosting.
I installed [https://github.com/Neoflow/ReverseProxy-PlausibleAnalytics](https://github.com/Neoflow/ReverseProxy-PlausibleAnalytics) on a subdomain stats.mysite.com
But I got an **404 error** on **/js/plausible.outbound-links.js** 
I whitelisted **/js/plausible.outbound-links.js** and the issue disappeared. 
==>But this means a **plausible.*.js** script is called. Which is very easy to notice for adblockers.

###My questions : 

- Will adblockers block this script no matter how it's called (because they rely on other footprints than the filename) ?
- What is the "official" URL ? plausible. or script. ? (both URLs deliver the file)
- In short : Should I investigate further ? Or should I just whitelist all the plausible.* script files ?

Thanks a lot !

**Comment by metmarkosaric** [comment]
thanks for the kind words @stremshs! you should call the file name script.js. it works the same way as plausible.js. in our proxy instructions, we recommend script.js as plausible.js will be blocked simply because of plausible in the file name

**Reply by stremshs** [reply]
Thanks for your quick answer !
The thing is : 
I'm not calling any script directly in the code as I use the Wordpress plugin, which is configured this way :
https://ibb.co/ncwXK2t
So my guess is that a script.js must be called, that calls the plausible.outbound-links.js file, but I'm no technical enough to reverse-engineer this supposed behavior.

**Reply by metmarkosaric** [reply]
i see. you won't be able to use our WP plugin for a proxy at the moment. "custom domains" as you see them in the plugin are no longer recommended either. we're hoping to remove the custom domain and introduce a proxy in the next major version of our WP plugin. for now, you'll need to manually integrate Plausible without a plugin for the proxy to work

**Reply by stremshs** [reply]
OK, thanks for the explanation !
It was quite convenient thought to exclude all admins from traffic.
I need to install another exclusion way then.
Can't wait to try the future major version of the plugin !
Cheers, keep on the good work !
